### PR TITLE
Edit child friendly page to read better

### PR DESCRIPTION
**Pull request by Sonosus**
I edited some clauses on the child friendly modes page to simplify them and make them easier to read.

### CODE REVIEW

File: docs/Child-friendly-EmulationStation.md [hunk 1]
```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
 ***
 
 # UI modes
-To allow different levels of system protection, user modes were introduced with different sets of functionalities. These modes are Full, Kiosk and Kid. Generally speaking, kids do not need to edit system settings, nor will visiting friends. The full set of options also complicates the UI, and the kid and kiosk modes hide these when they are not needed.
+To allow different levels of system protection, user modes have been introduced to allow the user to restrict certain settings and files from use. The three new modes are Full, Kiosk and Kid. In general, visiting friends and children do not need to modify system settings. The full set of options also complicates the UI, and the kid and kiosk modes hide these for a simpler experience.
```
Comment (cmitu) [inline docs/Child-friendly-EmulationStation.md:16]: There are only 2 'new' modes, the _Full_ mode is the default/built-in UI.

File: docs/Child-friendly-EmulationStation.md [hunk 2]
```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
...
 
 Setting UI mode to Full will give you the default ES user interface. This includes options such as CONFIGURE INPUT, UI SETTINGS, SCRAPER, but also EDIT THIS GAMES METADATA, and many other menu items that allow you to configure the system.
 
-Kiosk and Kid mode expose fewer of the menu options of Emulation station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot), with the exception of setting favorites. In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, even this functionality is removed.
+Kiosk and Kid mode expose fewer of the menu options of Emulation Station, only showing menu items that do not affect the system configuration (i.e. do not survive a reboot). In Kiosk Mode, the user is still able to toggle the favorite status of a game. In Kid-mode, this functionality is removed.
```
Comment (cmitu) [inline docs/Child-friendly-EmulationStation.md:20]: _Kid-mode_ should be _Kid mode_, for consistency.

File: docs/Child-friendly-EmulationStation.md [hunk 3]
```diff
@@ -13,23 +13,23 @@ These modes allow administrators to lock down (part of) the system, so that naiv
...
 
-In addition to this, the Kiosk and Kids modes allow for filtering of the game lists to remove unwanted items from the view, without actually removing the files from the system. The Kiosk mode only shows items in the game lists that are not explicitly set to be hidden (aka blacklisting). The Kid only shows items which are explicitly selected to be kid-friendly (aka whitelisting). Items which are set to be both `hidden` and `kidgame`, will be visible while in UI mode `kid`, but not while in UI mode `Kiosk`.
+In addition to this, the Kiosk and Kid modes allow for filtering of the game lists to hide unwanted games from view. This does not remove any files from the system. Kiosk mode only shows items in the game lists that are not explicitly set to be hidden (aka blacklisting). The Kid only shows items which are explicitly selected to be kid-friendly (aka whitelisting). Items which are set to be both `hidden` and `kidgame`, will be visible while in UI mode `kid`, but not while in UI mode `Kiosk`.
```
Comment (cmitu) [inline docs/Child-friendly-EmulationStation.md:22]: _The Kid only_ should perhaps be _The Kid mode only_.

**Review by cmitu — COMMENTED** [review]
Thank you for the edits.
I've left a couple of comments on the modifications, if you wish to add them to the modifications. Otherwise looks fine.

File: docs/Child-friendly-EmulationStation.md (hunk 3 above)
Comment (Sonosus) [reply docs/Child-friendly-EmulationStation.md]: Fixed in 1a9b8410db3f8f50cc44aebb5353dbc80f11b19d

File: docs/Child-friendly-EmulationStation.md (hunk 2 above)
Comment (Sonosus) [reply docs/Child-friendly-EmulationStation.md]: Resolved in 96f3dbbcb02027848cc006c951f7c0a5e8526ad9

File: docs/Child-friendly-EmulationStation.md (hunk 1 above)
Comment (Sonosus) [reply docs/Child-friendly-EmulationStation.md]: Good catch, fixed with 647c65e998bdc6c4ab84289e755dc773d8313aa7

**Review by cmitu — APPROVED** [review]

**Comment by cmitu** [comment]
@Sonosus thank you for the changes.

**Pull request merged**